general-concepts/ebuild-revisions/text.xml

```diff
@@ -98,8 +98,11 @@ of thumb could be used as a guideline:
     adding a missing build-time dependency that caused a build failure,
   </li>
   <li>
-    adding a new USE flag or removing an existing one (since change
-    in USE flags is going to trigger <c>--changed-use</c> rebuild),
+    adding a new USE flag (only where the functionality was hard-disabled in
+    the build system before) or removing an existing one (only where the
+    functionality is now disabled entirely, rather than always being enabled)
+    since the change in USE flags is going to trigger <c>--changed-use</c>
+    rebuild,
```

Comment:
Maybe it would be clearer to split this into separate items for adding and removing?

Also, [some say](https://thewritelife.com/when-to-use-parentheses/) that putting important information in parentheses isn't good style.

Reply:
Yeah, I think you're right. And the parentheses indeed should be for elaboration, not for further conditions.

Thanks, I'll give that a go.